09-schema/00-overview.md

```diff
@@ -6,33 +6,49 @@ permalink: /docs/schema/overview
 
 ## Why Use a Schema?
 
-Schema isa a mean to address the problems of managing and handling unstructured or loosely structured data.
+Schema isa a means to address the problems of managing and handling unstructured or loosely structured data.
 
 ![Unstructured problems](/docs/images/schema/unstructured-problems.png)
 
-The common problems we encounter when dealing with unstructured or loosely structured data are :
-- The problem of data integrity
-As the data is weakly tied to any particular structure it is hard or even impossible to control the state and validity of the data. As a result we have no guarantees on data consistency and validity.
-- The problem of data accessibility and retrieval
-With the lack of any high-level structure comes the lack of possibility to query the data meaningfully. This is either because our data structure is too low level to express complex queries or that handling the complexity of such queries becomes a problem.
+The common problems we encounter when dealing with unstructured or loosely structured data are:
+- Integrity
```

Comment:
I wasn't clear before, I think this format would be 👌 :
**Integrity** -- When data is weakly tied to any particular structure it is hard or even impossible to control the state and validity of the data.
.
.
.